### ISSUE TITLE
Problem about the demo.py

### ISSUE THREAD

**Issue by SWRDFK** [layumi/Person_reID_baseline_pytorch#138]
Hi, author, thank you for your excellent work on visualization. In the demo.py, we choose a query image by this code (`query_path, _ = image_datasets['query'].imgs[i]`), and the ID in the "query" set is right, but the output root of this image is from "bounding_box_test" set which is wrong. 
![图片](https://user-images.githubusercontent.com/40492484/58924895-277f7300-8778-11e9-95a4-963a3b01fc22.png)

In other word, the query image in "show.png" dismatch the real images as the following picture.
![图片](https://user-images.githubusercontent.com/40492484/58925020-8c3acd80-8778-11e9-9abe-832acbcebaaa.png)

**Comment by layumi** [comment]
Could you change the root to `Market/pytorch` not `bounding_box_test`

**Comment by SWRDFK** [comment]
Actually, I directly write the root into the dataset.ImageFolder, and don't know why ['query'] get the gallery image.
![图片](https://user-images.githubusercontent.com/40492484/58925549-89d97300-877a-11e9-900b-25726b3adfd7.png)

**Comment by layumi** [comment]
Do you have  `/home/Market/pytorch` ? Try this folder.

**Comment by SWRDFK** [comment]
Yes, I have tried. The first one is the 'query' image and the rest are gallery images. It's so strange that the query image is still in the gallery set.

![图片](https://user-images.githubusercontent.com/40492484/58925894-cb1e5280-877b-11e9-855c-1377c02f9966.png)

**Comment by layumi** [comment]
Have you change any codes?

**Comment by SWRDFK** [comment]
Nothing except this root. Could you get the right output of query image?

**Comment by Vinodini96** [comment]
Hi, Anyone please gave the line for market 1501 dataset, I don't know how to download it. The link in readme does not work!

**Comment by ZZH2950228** [comment]
where is the pytorch_result.mat, i can not find it

**Comment by layumi** [comment]
Hi @ZZH2950228 
You have to download the pretrained model and run `test.py` first.

**Comment by layumi** [comment]
@Vinodini96 
Hi, I updated the download link several days ago. Hope it could help you.
